ngViewer/index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html ng-app="ngViewer">
+<head>
+
+    <meta charset="utf-8">
+
+    <!--CSS-->
+    <link rel="stylesheet" type="text/css" href="css/bootstrap.min.css"/>
+    <link rel="stylesheet" type="text/css" href="css/bootstrap-theme.min.css"/>
+    <link rel="stylesheet" type="text/css" href="font-awesome/css/font-awesome.min.css"/>
```

Comment:
Please don't vendor in unused source code.

How can grunt.js (or some other task-runner) be used with package.json file for setting up the environment? I am asking b/c it isn't a good practice to have dependencies committed in version-control.